crates/holochain_websocket/src/lib.rs

```diff
@@ -688,13 +729,59 @@ impl WebsocketListener {
         Ok(Self {
             config,
             access_control,
-            listener,
+            listener: Box::new(listener),
+        })
+    }
+
+    /// Bind a new websocket listener on the same port using a v4 and a v6 socket.
+    pub async fn dual_bind(
+        config: Arc<WebsocketConfig>,
+        addr_v4: impl Into<SocketAddr>,
+        addr_v6: impl Into<SocketAddr>,
+    ) -> Result<Self> {
+        let access_control = Arc::new(config.allowed_origins.clone().ok_or_else(|| {
+            Error::other("WebsocketListener requires access control to be set in the config")
```

Comment:
If it's called `allowed_origins` this is clearer to me
```suggestion
            Error::other("WebsocketListener requires allowed origins to be set in the config")
```

Reply:
Good spot, updated